### PR TITLE
✨ Add service account support

### PR DESCRIPTION
**Pull request by bestak**
Merge after: https://github.com/AckeeCZ/Spreadsheet-Localizer-Plugin/pull/22

## Summary
Add support for fetching spreadsheets via service account authorization.

### CODE REVIEW

File: core/src/test/kotlin/cz/ackee/localizer/plugin/core/util/FileUtils.kt [hunk 1]
```diff
@@ -0,0 +1,12 @@
+package cz.ackee.localizer.plugin.core.util
+
+import java.io.File
+
+object FileUtils {
```
Comment (davidbilik) [inline core/src/test/kotlin/cz/ackee/localizer/plugin/core/util/FileUtils.kt:5]: hmm, i 'd probably remove the enclousing object and just define a public function. I think this is a relict from java world, kotlin code rarely uses this

**Review by davidbilik — APPROVED** [review]
Great job! Looking forward to test this

**Comment by bestak** [comment]
@davidbilik I deleted the other branch and the pull request closed itself preventing me to do any editing. Do you have the permissions to reopen the pull request and change the target branch to `master` please 🙏 ? Or should I create a new pull request? (Why is github like this 😭 )

**Comment by davidbilik** [comment]
It looks like you have to create a new PR 😞 dont know why it behaves like this

**Pull request closed without merging**